Kitodo/src/main/java/org/kitodo/services/file/FileService.java

```diff
@@ -770,8 +773,8 @@ public URI getSourceDirectory(Process process) {
      *
      * @return the URI to the temporal directory.
      */
-    public URI getTemporalDirectory() {
-        return Paths.get(ConfigCore.getParameter("tempfolder", "/usr/local/kitodo/temp/")).toUri();
+    public URI getTemporaryDirectory() {
+        return Config.getUri(Parameters.DIR_TEMP);
```

Comment:
Why did you remove the default value if this parameter is not defined?

Reply:
Because it does usually not work, or even creates merry havoc on the running system. On Windows, the default temp folder is `C:\Windows\Temp`. With this default, either something does just not work and you dont know why, or it may arbitrarily create folders `\usr\local\kitodo\temp` on whatever is your "current drive". On Linux, the temp folder usually is `/tmp/`. People hould get their config files right, rather than relying onto magic default values here. If we provide a built-in default, it must also be a correct one for Windows.